RuntimePerformanceTests/Benchmarks/BuildingSimulation/Benchmark.swift

```diff
@@ -0,0 +1,23 @@
+import Benchmark
+import _Differentiation
+
+let benchmarks: @Sendable () -> Void = {
+    Benchmark(
+        "BuildingSimulation - forward",
+        configuration: .init(metrics: [.wallClock, .mallocCountTotal])
+    ) { benchmark in
+        var simParams = SimParams(startingTemp: 33.3)
+        benchmark.startMeasurement()
+        blackHole(fullPipe(simParams: simParams))
+    }
+
+    Benchmark(
+        "BuildingSimulation - gradient",
+        configuration: .init(metrics: [.wallClock, .mallocCountTotal])
+    ) {benchmark in
+        var simParams = SimParams(startingTemp: 33.3)
+        benchmark.startMeasurement()
+        blackHole(gradient(at: simParams, of: { fullPipe(simParams: $0) } ))
```

Comment:
the initialization should affect the measurement I'd rather have this in one line I think unless I'm missing something.
so `gradient(at: SimParams(startingTemp: 33.3), of: { fullPipe(simParams: $0) } ))

Reply:
same on the first benchmark

Reply:
Ah I thought for a second it only contained a single value. Nvm in that case.